### ISSUE TITLE
Aspect Ratio: Height and Width are swapped

### ISSUE THREAD

**Issue by FFouetil** [Stability-AI/StableSwarmUI#20]
Hello, just noticed a bug that probably got past tests, as most people use 1:1 ratios.
![image](https://github.com/Stability-AI/StableSwarmUI/assets/11708007/27379204-0ec1-4bca-986a-644d21de60ee)

The output image uses the input dimensions, but vertical and horizontal axes are swapped.
(Also, it would be nice to have an option to use the source image dimensions when doing img2img)

Note: I'm using the ComfyUI local backend, in case it matters.

**Comment by mcmonkey4eva** [comment]
Er... that doesn't happen when I test it. Can you provide some more context about how you got to this bug?
Do you have a custom comfy workflow, or...?

**Comment by midihex** [comment]
can confirm, new install about 3 hours ago, with comfy backend, no custom workflow, SDXL1.0 Base & Refiner (With fixed internal vae) it took a few attempts at swapping the aspect ratios back and forth before it would pay attention, Also at the same time I noticed that while it was saying that the refiner was running on the output images, the single progress bar didn't reflect that. When it finally started using the aspect ration correctly It also showed the individual base and refiner progress bars - only the aspect ratio was changed between it working and not working if I recall.

**Comment by FFouetil** [comment]
After posting here, there's one instance where it worked correctly, and
then it bugged again, but at the moment I wasn't paying attention to
eventual parameter changes.
I tried to reproduce the conditions to compare but couldn't give it a lot
of time (life stuff occured).
IIRC, the moment it accidentally worked, I was playing around with some
toggles, including the refiner (which may have wrong inputs?).

I made sure to not use any custom node graph, but the ComfyUI I used was a
previously installed standalone, in which I did have a custom graph (Swarm
loaded the default one in another tab so I'm not sure it's related)

I'll try to dig a bit more a little later in the evening
